### PR TITLE
64 auto reload dev server

### PR DESCRIPTION
**Pull request by ductiletoaster**

### CODE REVIEW

File: packages/pixeloven-webpack-compiler/README.md [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+```
+
+or using yarn:
+
+```sh
+yarn add @pixeloven/webpack-compiler
+```
+
+## Usage
+> Note It is required that we provide a "client" and "server" configuration provide as an array.
```
Comment (jimmynotjim) [inline packages/pixeloven-webpack-compiler/README.md]: Nit: second "provide" is redundant and could maybe use clarity by adding "as a single array"

File: packages/pixeloven-webpack-compiler/jestrc.json [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+    "json",
+    "jsx",
+    "ts",
+    "tsx"
+  ],
+  "coveragePathIgnorePatterns": [
+    "/node_modules/",
+    "/test/"
+  ],
+  "coverageThreshold": {
```
Comment (jimmynotjim) [inline packages/pixeloven-webpack-compiler/jestrc.json]: Why `0`s here?

File: packages/pixeloven-webpack-config/src/configs/webpack/server.ts [hunk 3]
```diff
@@ -40,12 +40,13 @@ const config = (env: NodeJS.ProcessEnv): Configuration => {
     /**
      * Define entrypoint(s) for sever
      */
-    const entry = removeEmpty([
-        ifProduction(
-            resolvePath("src/server/index.ts"),
-            resolvePath("src/server/webpack.ts"),
-        ),
-    ]);
+    // const entry = removeEmpty([
```
Comment (jimmynotjim) [inline packages/pixeloven-webpack-config/src/configs/webpack/server.ts]: Should this get a TODO?

File: packages/pixeloven-webpack-compiler/jestrc.json (hunk 2 above)
Comment (ductiletoaster) [reply packages/pixeloven-webpack-compiler/jestrc.json]: I don't know how to test this part yet. So for now this is going to be 0 coverage

File: packages/pixeloven-webpack-compiler/README.md (hunk 1 above)
Comment (ductiletoaster) [reply packages/pixeloven-webpack-compiler/README.md]: updated

**Pull request merged**